src/Skills/Skills.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Skill from '../Skill/Skill';
+
+const Skills = () => {
+  const [skillsState] = useState({
+    skills: [
+      {title: 'JavaScript', description: 'Learn it'},
+      {title: 'Java', description: 'Keep learning it'}
+    ]
+  });
```

Comment:
Will be using GraphQL to get the skills when LOOFAH-3 is done, felt that this was the most simple way to get something  together to start with.